docs/reference/commandline/run.md

```diff
@@ -262,6 +262,13 @@ https://docs.docker.com/engine/installation/binaries/#/get-the-linux-binary)),
 you give the container the full access to create and manipulate the host's
 Docker daemon.
 
+On Windows, the paths must be specified using Windows-style semantics. Windows
+containers do not support the target of a volume or bind-mount being pre-
+populated in the container image.
```

Comment:
As opposed to Linux, where you can splat existing contents:

![image](https://cloud.githubusercontent.com/assets/10522484/20581470/e80b3e74-b18d-11e6-91f2-a8c85f85b577.png)